hw-2/server.ts

```diff
@@ -0,0 +1,64 @@
+import express = require('express');
+import {UsersStorage} from './storage/users';
+import bodyParser = require('body-parser');
+import {ContainerTypes,
+        // Use this as a replacement for express.Request
+        ValidatedRequest,
+        // Extend from this to define a valid schema type/interface
+        ValidatedRequestSchema,
+        // Creates a validator that generates middlewares
+        createValidator} from 'express-joi-validation';
+import {User, userSchema} from './models/user';
+
+interface UserRequestSchema extends ValidatedRequestSchema {
+  [ContainerTypes.Body]: User;
+}
+
+const PORT = 3000;
+
+const app = express();
+app.use(bodyParser.json());
+
+const validator = createValidator();
+const usersStorage = new UsersStorage();
+
+app.get('/users', (req, res) => {
+  res.json(usersStorage.getUsers());
```

Comment:
please move all the functionality from route file to controller file, to have one separate file with api structure code and another with functionality